subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1479,6 +1479,40 @@ static int cmd_wifi_ps_wakeup_mode(const struct shell *sh, size_t argc, char *ar
 	return 0;
 }
 
+static int cmd_wifi_set_rts_threshold(const struct shell *sh, size_t argc, char *argv[])
+{
+	struct net_if *iface = net_if_get_first_wifi();
+	unsigned int rts_threshold = -1; /* Default value if user supplies "off" argument */
```

Comment:
shoudln't this be `signed` or is it intentional?

Reply:
It's intentional to keep the default value (-1).